admin/app/views/spree/admin/digital_assets/create.turbo_stream.erb

```diff
@@ -0,0 +1,5 @@
+<%= turbo_render_alerts %>
```

Comment:
@Vegann let's do a redirect after create - https://discuss.hotwired.dev/t/redirect-after-turbo-stream-response/2303/10 and do not use turbo stream if save is succesful